sklearn/linear_model/coordinate_descent.py

```diff
@@ -1084,7 +1084,10 @@ def fit(self, X, y):
             to avoid unnecessary memory duplication. If y is mono-output,
             X can be sparse.
 
-        y : array-like, shape (n_samples,) or (n_samples, n_targets)
+        y : array-like, shape 
```

Comment:
This will render strangely in HTML. Rather, don't say shape here and put the shape in the description below.

Reply:
I believe it should be solved now